mtest/libmscore/compat206/lidemptytext.mscx

```diff
@@ -153,10 +153,6 @@
         </VBox>
       <Measure number="1">
         <stretch>0.9</stretch>
-        <Clef>
-          <concertClefType>G</concertClefType>
-          <transposingClefType>G</transposingClefType>
-          </Clef>
```

Comment:
Why did you remove this clef? It seems to be removed from the source file, not from the reference?

Reply:
Hmmm, I'll look into that...

Reply:
Guess you'd need to add them to the ref file instead?

Reply:
No it's not that simple. There isn't supposed to be changes in these two files, so I guess I made some pretty harmful deletions back then.